bazel/emscripten_deps.bzl

```diff
@@ -40,15 +61,33 @@ filegroup(
         "bin/wasm-emscripten-finalize{bin_extension}",
         "bin/wasm-ld{bin_extension}",
         "bin/wasm-opt{bin_extension}",
-   ] + glob(["emscripten/node_modules/**"]),
+        "bin/wasm-metadce{bin_extension}",
+        ":emcc",
+    ] + glob(
+        include = [
+            "emscripten/cache/sysroot/lib/**",
+            "emscripten/node_modules/**",
+            "emscripten/src/**",
```

Comment:
emcc.py depends on stuff in `tools` so I'm surprised not to see it here.  

I think `tools` is needed be all steps (since it contains a lot of shared python code that all entry points depend on) so I would imagine it would need to go in some kind of "common" fileset?

Reply:
Oh I see `emcc` is that name for this shared set... In that case you can remove `emcc.py` from this set?

Reply:
Done.